src/Akka.Streams.Kafka/reference.conf

```diff
@@ -135,7 +136,7 @@ akka.kafka.committer {
     max-interval = 10s
     
     # Set parallelism for async committing
-    parallelism = 1
+    parallelism = 100
```

Comment:
Set the parallelism for committing equal to 100 in-flight commit attempts at a time. These will get aggregated together by the `KafkaConsumerActor` and consolidated to highest-offset-per-partition, so there's no consistency issues there. This just allows the stream to demand more while commits are in-process.